interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -41,6 +41,7 @@
 #include <cstdio>
 
 #include <memory>
+#include <sstream>
```

Comment:
Oups, I forgot to remove that. It's not used indeed.
Removed here : https://github.com/Y--/root/commit/83e6fc71efef78ce45d7129550e6c995afbac3a1